src/mlpack/tests/serialization.hpp

```diff
@@ -43,9 +43,9 @@ void TestArmadilloSerialization(arma::Cube<CubeType>& x)
 
     try
     {
-      o << BOOST_SERIALIZATION_NVP(x);
+      o << CEREAL_NVP(x);
     }
-    catch (boost::archive::archive_exception& e)
+    catch (cereal::Exception& e)
```

Comment:
I wonder if it would be better to just let the exception throw here, and then it would be propagated up to the tests, where some actual good output is shown, instead of just a report that `success == false`.  What do you think?

Reply:
Yes, of course, if the exception is giving more details it would be a much better answer than the `success == false` in which we already know that.
I think it is the case right now?

Reply:
Yeah, like we saw this morning, removing all the `try`/`catch` blocks gives way better output.  So I think we should adjust the code accordingly. :+1: